src/redux/books/books.js

```diff
@@ -0,0 +1,39 @@
+const ADD_BOOK = 'ADD_BOOK';
+const REMOVE_BOOK = 'REMOVE_BOOK';
```

Comment:
Defines `ADD_BOOK` and `REMOVE_BOOK` as action types ✔

Reply:
Please be sure that your actions types follow the ducks pattern 🤓 



     const ADD_BOOK = bookstore/books/ADD_BOOK